### PR TITLE
Update v2.markdown

### PR DESCRIPTION
**Pull request by ChrisbZx**
Torque FAQ fix - origination fee added

### CODE REVIEW

**Review by RomanHiden — APPROVED** [review]
looks good

File: packages/torque-website/v2.markdown [hunk 1]
```diff
@@ -350,7 +350,7 @@ permalink: v2
                             <span class="accordion-position"></span>
                         </h4>
                         <div class="accordion-content" aria-hidden="false">
-                            <p>There are no fees involved with opening the loan. You just pay the interest.</p>
+                            <p> When opening a loan there is a 0.09% origination fee on the principle. Interest is also paid for the life of the loan. .</p>
```
Comment (RomanHiden) [inline packages/torque-website/v2.markdown:353]: you have spacing in front and extra dot in the end

**Review by RomanHiden — CHANGES_REQUESTED** [review]

**Comment by Kharabet** [comment]
BZRX, LEND backgrounds are transparent

**Review by ChrisbZx — COMMENTED** [review]
Approved

**Review by RomanHiden — APPROVED** [review]
looks good

**Review by Kharabet — APPROVED** [review]

**Pull request merged**